pkg/build/builder/cmd/builder.go

```diff
@@ -41,22 +41,28 @@ func run(builderFactory factoryFunc) {
 		authcfg     docker.AuthConfiguration
 		authPresent bool
 	)
+	output := true
 	if len(build.Parameters.Output.DockerImageReference) == 0 {
 		if build.Parameters.Output.To != nil {
```

Comment:
No, if Output.To==nil we'll fall through to "output=false" but still run the build, which is as desired.